test/e2e_tests/specs/Block/block.spec.ts

```diff
@@ -104,222 +103,241 @@ test.describe('Block', () => {
+        // Step 6: User B writes message to User A
+        await userBPages.conversation().sendMessage('Message after being unblocked');
+        // Step 7: User A receives message from User B
+        await userAPages.conversationList().openConversation(userB.fullName);
+        await expect(userAPages.conversation().messages).toHaveCount(2);
+      },
+    );
+
+    test(
+      'Verify you cannot add a person who blocked you to a group chat 0',
+      {tag: ['@TC-143', '@regression']},
+      async ({createPage}) => {
+        const userAContext = await PageManager.from(
+          createPage(withLogin(userA), withConnectedUser(userC), withConnectionRequest(userB)),
+        );
+        const userAPageManager = userAContext.webapp;
+        const {pages: userAPages, modals: userAModals, components: userAComponents} = userAPageManager;
+
+        const conversationName = 'Group Conversation';
+
+        // Preconditions: User B accepts the connection request
+        const userBContext = await PageManager.from(createPage(withLogin(userB)));
+        const userBPages = userBContext.webapp.pages;
+        await userBPages.connectRequest().clickConnectButton();
+
+        // Step 1: User B blocks User A
+        await test.step('User B blocks User A', async () => {
+          await userAPages.conversationList().openConversation(userC.fullName);
+          await blockUserFromConversationList(userBContext, userA);
+        });
+
+        // Step 2: User A wants to add B to a group chat after being blocked by User B
+        await test.step('Users A tries to add B to a group', async () => {
+          await userAComponents.conversationSidebar().isPageLoaded();
+          await userAPages.conversationList().clickCreateGroup();
+          await userAPages.groupCreation().setGroupName(conversationName);
+          await userAPages.startUI().selectUsers([userB.username]);
+          await userAPages.groupCreation().clickCreateGroupButton();
+
+          // Step 3: Modal 'modalConversationNotConnectedMessageOne' is visible
+          await expect(userAModals.acknowledge().modal).toBeVisible();
+        });
+      },
+    );
+
+    test(
+      'Verify you can block a user you sent a connection request from conversation list 0',
+      {tag: ['@TC-144', '@regression']},
+      async ({createPage}) => {
+        const userAContext = await PageManager.from(createPage(withLogin(userA), withConnectionRequest(userB)));
+
+        // Preconditions: User B does not accept connection request
+        const userBPages = (await PageManager.from(createPage(withLogin(userB)))).webapp.pages;
+
+        // Step 2: User A blocks User B from conversation list
+        await blockUserFromConversationList(userAContext, userB);
+        // Step 3: User B does not see Connection Request from User A
+        await expect(userBPages.conversationList().getConversationLocator(userA.fullName)).not.toBeVisible();
```

Comment:
This test passes even if the action above is commented out. Please make sure the test can fail, e.g. by making sure the conversation is visible for userB before A blocks him.